tests/components/fyta/test_init.py

```diff
@@ -26,13 +26,24 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.mark.parametrize(
+    "expiration",
+    [
+        EXPIRATION,
+        EXPIRATION_OLD,
+    ],
+)
```

Comment:
Why do we do this?

Reply:
To test if the renew_authentication in coordinator works (i.e. if new access_token with new expiration date is returned)

Reply:
-> coverage for L64-L72 in coordinator.py

Reply:
Should we make that a separate test? It's now added without context, and nowhere we really describe what it actually does.

With the separate test, we can also test that we call the right function on the client

Reply:
Or should I add a comment on L32/33 to explain?

Reply:
I mean, parametrizing is nice, but in this case it's losing context. I think the test for the reauth can be different than this test. For example, test if the token really got updated in the config entry etc.

This feels more like touching code for just touching than actually testing if the code does what it does